lib/adapters/apply-edit-adapter.ts

```diff
@@ -90,7 +90,7 @@ export default class ApplyEditAdapter {
     const startRow = edit.oldRange.start.row
     const startCol = edit.oldRange.start.column
     const lineLength = buffer.lineLengthForRow(startRow)
-    if (lineLength == null || startCol > lineLength) {
+    if (startCol > lineLength) {
```

Comment:
We should fix the original types then. Writing code based on the wrong types or docs is not a good idea!

Reply:
Sure. My point is, we _really_ should verify we don't accidentally break stuff by these linter-driven changes. Regrettably, TS isn't Haskell, so it can lie about types, especially when definitions are written by fallible humans.